### PR TITLE
added card details + some code cleanup

### PR DESCRIPTION
**Pull request by xshi0603**
modified backend in first commit (someone double check that 😄)

### CODE REVIEW

File: api/routes/favorites.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ router.get("/", async (req, res) => {
  */
 router.post("/", async (req, res) => {
   try {
-    const { orgName, orgId, orgAddress, userId } = req.query;
+    const { orgName, orgId, orgAddress, userId, tagLine } = req.query;
```
Comment (sdhani) [inline api/routes/favorites.js:38]: ✨

File: client/src/components/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { Card as BootstrapCard } from 'react-bootstrap'
+import { Favorite } from '../components/Favorite'
+import "../styles/BigCard.css"
+
+export const Card = ({name, ein, isFavorited, organization}) => {
+  const [mailingAddress, setMailingAddress] = useState({city: "LOADING", stateOrProvince: "", postalCode: ""}); //used so that you don't try to render data before it is loaded
+
+  useEffect(() => {
+    if (organization.length !== 0) {
+      setMailingAddress({city: organization.mailingAddress.city, stateOrProvince: organization.mailingAddress.stateOrProvince, postalCode: organization.mailingAddress.postalCode});
```
Comment (sdhani) [inline client/src/components/Card.jsx:11]: 👍

**Comment by wongjessica** [comment]
lgtm

File: api/routes/favorites.js (hunk 1 above)
Comment (wongjessica) [reply api/routes/favorites.js:38]: ✨

**Review by wongjessica — APPROVED** [review]
lgtm

**Pull request merged**